cfme/tests/containers/test_containers_properties.py

```diff
@@ -18,6 +21,12 @@
 pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
 
 
+def randomized_names_of(tbl, count):
+    names = [r.name.text for r in tbl.rows()]
+    random.shuffle(names)  # inplace
+    return names[:min(count, tbl.row_count())]
```

Comment:
Look [above](https://github.com/ManageIQ/integration_tests/pull/4082#discussion-diff-98388485R46) my friend, I used this one but @RonnyPfannschmidt ask me to create this function.